numpy/core/src/multiarray/nditer_pywrap.c

```diff
@@ -820,6 +827,13 @@ npyiter_init(NewNpyArrayIterObject *self, PyObject *args, PyObject *kwds)
         goto fail;
     }
 
+    for (iop = 0; iop < nop; ++iop) {
+        if (op_flags[iop] & NPY_ITER_READWRITE) {
+            self->needs_context_manager = 1;
+        }
+
```

Comment:
extra blank line here

Reply:
also, all lines related to `needs_context_manager` can be removed now.

Reply:
fixed